web_external/vue/views/AnnotationTool.vue

```diff
@@ -0,0 +1,280 @@
+.annotation-tool-container.container-fluid
+  .isic-page.row
+    .isic-pane.col-sm-3
+      .header
+        .info-group
+          span.title ISIC Annotation Tool
+          a.pull-right(href='/') Return home.
+        .info-group
+          div(v-if='image')
+            span #[b Image:]&nbsp;{{ image.name }}
+        .info-group
+          div(v-if='user')
+            span #[b User:]&nbsp;{{ user.name() }}
+        .info-group
+          .dropdown
+            button.btn.btn-warning.dropdown-toggle(
+              type='button',
+              data-toggle='dropdown'
+            )
+              span Flag Image&nbsp;
+              span.caret
+            ul.dropdown-menu
+              li
+                a(@click='setFlagStatus("ok")') Unflagged
+              li
+                a(@click='setFlagStatus("phi")') Contains PHI
+              li
+                a(@click='setFlagStatus("quality")') Low optical quality
+              li
+                a(@click='setFlagStatus("zoom")') Inadequate zoom factor
+              li
+                a(@click='setFlagStatus("inappropriate")') Clinically inappropriate
+              li.divider
+              li
+                a(@click='flag("other")') Other reason
+            span.flag-status
+              span #[b Flag Status:]&nbsp;{{ flagStatus }}
+      .annotation(v-if='questions.length > 0')
+        h3.annotation-header Global Features
+        annotation-tool-global-features
+      .annotation(v-if='questions.length > 0')
+        h3.annotation-header Local Features
+        .annotation-section
+          annotation-tool-local-features(
+            :activeFeatureId='activeFeatureId',
+            @featureActivated='onFeatureActivated',
+            @featureDeactivated='onFeatureDeactivated',
+            @displayFeature='onDisplayFeature',
+            @deleteFeature='onDeleteFeature'
+          )
+      .annotation-section
+        .btn-group
+          template(v-if='showReview')
+            button.btn.btn-info(
+              type='button',
+              @click='setShowReview(false)'
+            ) Continue editing
+          template(v-else)
+            button.btn.btn-info(
+              type='button',
+              @click='reset'
+            ) Reset current
+        .btn-group.pull-right
+          template(v-if='showReview')
+            button.btn.btn-success(
+              type='button',
+              @click='submitAnnotation'
+            ) Submit
+          template(v-else)
+            button.btn.btn-success(
+              type='button',
+              @click='setShowReview(true)'
+            ) Review
+    .isic-map.col-sm-9
+      annotation-tool-viewer.isic-map(
+        ref='viewer',
+        :image='image'
+      )
+</template>
+
+<script>
+import { createNamespacedHelpers } from 'vuex';
+
+import { getApiRoot } from 'girder/rest';
+import { getCurrentUser } from 'girder/auth';
+
+import AnnotationToolGlobalFeatures from './AnnotationToolGlobalFeatures.vue';
```

Comment:
Can we rename these classes and files to refer to `Questions` instead of `GlobalFeatures`?